src/main/java/com/simpligility/maven/plugins/android/standalonemojos/ManifestMergerMojo.java

```diff
@@ -372,7 +372,7 @@ public void manifestMergerV2() throws MojoExecutionException, MojoFailureExcepti
                   androidManifestFile,     // mainManifest
                   new ArrayList<File>(),   // manifestOverlays
                   manifestDependencies,    // libraries
-                  "",                      //featurename
+                  null,                      //featurename
```

Comment:
- https://github.com/Card-Forge/forge/pull/4977

Reply:
mvn clean install -DskipTests -e should work to build the jar

Reply:
Better Link would be to this: https://github.com/simpligility/android-maven-plugin/issues/814